principles/3_Privacy.md

```diff
@@ -0,0 +1,51 @@
+# Privacy
+
+## Data Minimization
+
+* The minimum amount of data possible shall be collected, for the purposes for which it is gathered. Data that is not used in the disclosure to healthcare authorities should not be collected unless it is essential to the purposes of the application
+* Users can choose how much data to share from symptom surveys
+
+## Consent
+* A member of the public (MoP) shall consent to data being collected or transmitted explicitly, that is it should not be implied by usage, or mixed with another consent
+* A MoP shall provide informed consent, that is they must be able to understand what they are consenting to and the purposes for which each category of data is gathered
+* Information about privacy shall be communicated in plain English, not legal, or technical language
+* Consent to collect data shall be separate from consent to disclose data, and include retention periods
+* Consent to collect each category of data (e.g. location, proximity) shall be separate
+* No communication initiated by 3rd parties shall be provided to a MoP, through the app, without their consent
+* No data shall be shared with 3rd parties, except healthcare authorities using the agreed mechanisms
+* The user of SafePaths shall be able to disable location tracking temporarily
+* Consent to disclose data to a HA shall be separate, at the time of disclosure, and contain a reference to the HA privacy policy
+
+## Transparency & Accountability
+
+* The privacy policy shall include the steps taken by the data collector to ensure the confidentiality, integrity, and quality of the data
+* The data controller, that is, who controls what data is collected and how it is stored and disclosed shall be identified clearly
```

Comment:
Suggested edit for clarity:
"It shall be identified clearly who controls what data is collected and how it is stored and disclosed."

Reply:
My readability nit here is mainly that "the data controller" and "identified clearly" are on opposite sides of the sentence.

Reply:
Perhaps "The privacy policy shall identify clearly both the person who controls the data that is collected and how the data is stored and disclosed. "

Reply:
I've updated in line with @skheinze proposal which I think is slightly clearer!